packages/react/src/hooks/useBreakpointValue.ts

```diff
@@ -6,10 +6,10 @@ import { useTheme } from './useTheme';
 /**
  * [📖 Docs](https://ui.docs.amplify.aws/react/theming/responsive#usebreakpointvalue)
  */
-export function useBreakpointValue<T>(
-  values: Record<string, T> | T[],
+export function useBreakpointValue(
```

Comment:
Removed the generic here as it our usage of this hook always takes in an argument of `Record<string, string>` and returns `string` (or `null`)

Reply:
I did some updates on the get function because I found that `number` is allowed for some props. But I am agree on removing generic. I will update my PR based on yours afterwards